### PR TITLE
fix(deleteFile): fixed delete file

### PR DESCRIPTION
**Pull request by simran142002**
### Functionality:
Users can now delete files in organisation page

### Solution:
made a separate method for delete and created the api file for it

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
1)ng serve
2)firebase emulators:start
3)worktez.com
4)my organization
5)document
6)upload and delete file

### CODE REVIEW

File: functions/model/librarian/tark/deleteFilesInOrg.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  const time = request.body.data.Time;
+  const orgFileDocumentName = request.body.data.OrgFileDocumentName;
+
+  let result;
+  let status = 200;
+
+  const promise = getOrgUseAppKey(appKey).then((orgDetail) => {
+      const orgDomain = orgDetail.OrganizationDomain;
+
+      const promise1 = getOrg(orgDomain).then((orgDoc) => {
```
Comment (viv9k) [inline functions/model/librarian/tark/deleteFilesInOrg.js:29]: I didn't get it we are getting org multiple times

File: functions/model/librarian/tark/deleteFilesInOrg.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable max-len */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable object-curly-spacing */
+/* eslint-disable no-unused-vars */
+
+const { updateFileToTask } = require("../lib");
+const { updateFileToOrg } = require("../lib")
```
Comment (Pendyala-thanvi) [inline functions/model/librarian/tark/deleteFilesInOrg.js:8]: const { updateFileToOrg } = require("../lib"). This statement is written twice. Remove one.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please connect over it for more clarity

File: functions/model/librarian/tark/deleteFilesInOrg.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+const { getOrg } = require("../../organization/lib");
```
Comment (Pendyala-thanvi) [inline functions/model/librarian/tark/deleteFilesInOrg.js:9]: getOrg, updateOrg, and getOrgUseAppKey are from a single file. get them all at once.

**Review by Pendyala-thanvi — COMMENTED** [review]
Do the mentioned changes.

**Review by Pendyala-thanvi — COMMENTED** [review]
resolve merge conflicts.

**Comment by viv9k** [comment]
Merge conflict not resolved properly and the delete fix was having all the changes from previous release

**Pull request closed without merging**